spec/content/automate/ManageIQ/System/Request.class/__methods__/order_ansible_playbook_spec.rb

```diff
@@ -132,6 +132,25 @@
     end
 
     it_behaves_like "order playbook"
+
+    context 'with no floating_ip_addresses' do
```

Comment:
@gmcculloug would be nice to have a spec where :floating_ip_addresses method is not defined (I think it's defined only for Cloud Vms objects)

or is that included in ```it_behaves_like "order playbook"``` ?

Reply:
The default VM in this spec is `FactoryGirl.create(:vm_vmware, :name => 'fred')` which would not have the `floating_ip_addresses` method defined.  Therefore all previous tests would be running under that condition: `"with no machine credential", "with machine credential", "with vm host"`

It would be better to have an explicit test for that in case someone changes the `vm` factory to be a cloud VM.  I'll add it.

Reply:
awesome, thank you